src/modules/configs/config.py

```diff
@@ -0,0 +1,48 @@
+import os
+from json import loads
+from typing import Union
+
+from jsonschema import validate, ValidationError
+
+from ingestation.modules.global_vars import APPLICATION_DIR
+
+
+def validate_json(conf: dict, schema_file) -> None:
+    validate(instance=conf, schema=getattr(schema_file, "schema"))
+
+
+class Conf:
+    """Class for parsing configuration files"""
+
+    def __init__(self, conf_path: Union[bool, str], conf_type: str, schema_file):
+        self.conf_type = conf_type
+        self.conf_path = self.get_conf_path(conf_path)
+        self.schema_file = schema_file
+
+    def parse_conf(self) -> Union[bool, dict]:
+        conf = self.load_conf()
+
+        try:
+            validate_json(conf, self.schema_file)
+        except ValidationError as ex:
```

Comment:
Raising exception is the preferred Python solution and I am not trying to undermine that. However, this application is targeting ingest Operators and (from my experience) they tend to be more on the user than developer side of things, so we should keep UX in mind. Best solution would be to create a child class in `modules.outputs` that would colour our exceptions and perhaps limit tracebacks in some cases (most users only read the last line anyway) - similar to `modules.outputs.loggers.ColourFormatter`. For sure this is a task for another MR (#44).

Reply:
you are right. let me think about it